output.go

```diff
@@ -0,0 +1,83 @@
+package goyek
+
+import (
+	"fmt"
+	"io"
+	"sync"
+)
+
+func writeLinef(w io.Writer, format string, a ...interface{}) {
+	_, _ = fmt.Fprintf(w, format+"\n", a...)
+}
+
+// Output contains the writers to communicate results.
+type Output struct {
+	// Standard output is for information that is expected from the executed tasks.
+	Standard io.Writer
+	// Messaging output is for any status information, such as logging or error messages.
+	Messaging io.Writer
+}
+
+// WriteMessagef prints the given format message and its arguments to the Messaging writer.
+// The result of this action is ignored.
+func (out Output) WriteMessagef(format string, a ...interface{}) {
+	writeLinef(out.Messaging, format, a...)
+}
```

Comment:
1. I am not sure that it is good to export this method.
2. Also it is not consistent that there is no parallel method for writing the primary/standard output.